### PR TITLE
Router move

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

File: modules/@angular/router-deprecated/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "skipTemplateCodegen": true
+  },
+  "compilerOptions": {
+    "baseUrl": ".",
+    "declaration": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "outDir": "../../../dist/packages-dist/router/",
```
Comment (IgorMinar) [inline modules/@angular/router-deprecated/tsconfig.json]: .../router-deprecated/

File: modules/@angular/router-deprecated/tsconfig-es2015.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "skipTemplateCodegen": true
+  },
+  "compilerOptions": {
+    "baseUrl": ".",
+    "declaration": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "module": "es2015",
+    "moduleResolution": "node",
+    "outDir": "../../../dist/packages-dist/router/esm",
```
Comment (IgorMinar) [inline modules/@angular/router-deprecated/tsconfig-es2015.json]: .../router-deprecated/esm

Comment (mhevery) [inline modules/@angular/router-deprecated/tsconfig-es2015.json]: you are better then compiler. :-) fixed

File: modules/@angular/router-deprecated/tsconfig.json (hunk 1 above)
Comment (mhevery) [inline modules/@angular/router-deprecated/tsconfig.json]: fixed

**Comment by mhevery** [comment]
merged 107016e

**Pull request closed without merging**